development/creds.example.env

```diff
@@ -1,35 +1,48 @@
-# nautobot_chatops
-SLACK_API_TOKEN=xoxb-...
-SLACK_SIGNING_SECRET=...
-# SLACK_SLASH_COMMAND_PREFIX=/
-MICROSOFT_APP_ID=...
-MICROSOFT_APP_PASSWORD=...
-WEBEX_TEAMS_ACCESS_TOKEN=...
-WEBEX_TEAMS_SIGNING_SECRET=...
-MATTERMOST_API_TOKEN="5qsffxoapt883qfdygbdgf17jy"
 POSTGRES_PASSWORD=notverysecurepwd
 REDIS_PASSWORD=notverysecurepwd
 SECRET_KEY=r8OwDznj!!dci#P9ghmRfdu1Ysxm0AiPeDCQhKE+N_rClfWNj
 NAUTOBOT_CREATE_SUPERUSER=true
 NAUTOBOT_SUPERUSER_API_TOKEN=0123456789abcdef0123456789abcdef01234567
 NAUTOBOT_SUPERUSER_PASSWORD=admin
-NAUTOBOT_TOWER_PASSWORD="password"
 
-# Cisco ACI Chatops Plugin Settings
+# = Chat Platforms ===================
+
+# - Mattermost -----------------------
+MATTERMOST_API_TOKEN="5qsffxoapt883qfdygbdgf17jy"
+
+# - Microsoft Teams ------------------
+# MICROSOFT_APP_ID="changeme"
+# MICROSOFT_APP_PASSWORD="changeme"
+
+# - Slack ----------------------------
+# SLACK_API_TOKEN="xoxb-changeme"
+# SLACK_APP_TOKEN="changeme"
+# SLACK_SIGNING_SECRET="changeme"
+
+# - Webex ----------------------------
```

Comment:
We went with Webex Teams when it was rebranded. Did Webex Teams become Webex again?

Reply:
In addition, I know it doesn't really matter, as Admins can change the nautobot_config.py to whatever they want, it will still cause issues when using an older creds.env for developers expecting WEBEX_TEAMS_ACCESS_TOKEN.

Reply:
Expecting the next ChatOps App version to be 2.0, I was taking into account this deprecation: https://github.com/nautobot/nautobot-plugin-chatops/blob/develop/docs/admin/install/webex_setup.md#deprecation-warning

`Webex Teams` brand doesn't seem to be advertised by Ciso, when using search tools.

The old `WEBEX_TEAMS_ACCESS_TOKEN` is directly consumed by `webexteamssdk` library, however `WEBEX_ACCESS_TOKEN` is read by the App `nautobot_config.py`, passed to App config, and then passed to the library as arguments. Seems to me better a bit better to use environment variables controlled by our App config to allow us some validations.

I would stay with `Webex` and emphasize this change in the documentation I'm preparing WDYT?

Reply:
Webex cleaned up:

- Renamed `WebEx` => `Webex`.
- Simplified reading App config.
  - Added `def get_app_config_part(prefix)` .
- Removed `webex_teams` related deprecation warnings and usage.
- Added `webex_msg_char_limit` App config entry.